e2e_tests/tests/fixtures/metric_maker/metric_maker.py

```diff
@@ -85,17 +85,17 @@ def from_trial(trial_inst: det.Trial, *args: Any, **kwargs: Any) -> det.TrialCon
     def pre_execute_hook(env: det.EnvContext, hvd_config: horovod.HorovodContext) -> None:
         pass
 
-    def train_for_step(self, step_id: int, batches_per_step: int) -> Dict[str, Any]:
+    def train_for_step(self, step_id: int, scheduling_unit: int) -> Dict[str, Any]:
```

Comment:
This is super nit-picky, but this shouldn't really be called `scheduling_unit`, which is a configuration-level constant.  It should be called `num_batches` or `batches_to_train` or something that indicates that this value is specific to this `train_for_step`.

That said, I care literally zero about this detail for the CallbackTrialController subclasses, which are exclusively used in testing.  I'm bringing it up in case there are other cases you know of which are more important.

Reply:
I looked for other cases of this that were more important and didn't notice any

Reply:
Yeah I saw this and more or less decided since it was in `e2e_tests` I wasn't _too_ concerned about it.